pkgs/top-level/all-packages.nix

```diff
@@ -1782,8 +1782,6 @@ with pkgs;
 
   fscan = callPackage ../tools/security/fscan { };
 
-  copier = callPackage ../tools/misc/copier { };
```

Comment:
This removes the attr from pkgs

Please add back:
```
copier = with python3.pkgs; toPythonApplication copier;
```

This will at least bring back the copier cli as a python application

Reply:
Thanks, done.